src/quic/node_quic_socket.cc

```diff
@@ -83,24 +83,23 @@ bool IsShortHeader(
 }  // namespace
 
 QuicPacket::QuicPacket(const char* diagnostic_label, size_t len) :
-    data_(len),
+    data_{0},
+    len_(len),
     diagnostic_label_(diagnostic_label) {
-  CHECK_LE(len, NGTCP2_MAX_PKT_SIZE);
+  CHECK_LE(len, NGTCP2_MAX_PKTLEN_IPV4);
```

Comment:
```suggestion
  CHECK_LE(len, MAX_PKTLEN);
```
and same for `-inl.h`?